Install/install.sh

```diff
@@ -0,0 +1,205 @@
+    install_python_packages
+
+    # install any additional libraries we need to build manually
+    build_python_libraries
+  fi
+
+  cd $LOCAL_DIR
+  echo -e "SlowMovie install complete"
+  echo -e "To test run $(tput setaf 3)python3 slowmovie.py$(tput sgr 0)"
+
+}
+
+function install_service(){
+  if [ -d "${LOCAL_DIR}" ]; then
+    cd $LOCAL_DIR
+    # install the service files and enable
+    sudo cp slowmovie.service /etc/systemd/system
+    sudo systemctl daemon-reload
+    sudo systemctl enable slowmovie
+
+    echo -e "SlowMovie service installed use $(tput setaf 3)sudo systemctl start slowmovie$(tput sgr 0) to test"
+  else
+    echo -e "$(tput setaf 1)SlowMovie repo does not exist, do that first$(tput sgr 0)"
+  fi
+
+  # go back to home
+  cd /home/pi
+}
+
+function uninstall_service(){
+  if [ -f "/etc/systemd/system/slowmovie.service" ]; then
+    # stop if running and remove service files
+    sudo systemctl stop slowmovie
+    sudo systemctl disable slowmovie
+    sudo rm /etc/systemd/system/slowmovie.service
+    sudo systemctl daemon-reload
+
+    echo -e "SlowMovie service is uninstalled"
+  else
+    echo -e "$(tput setaf 1)SlowMovie is not installed, nothing to clean up$(tput sgr 0)"
+  fi
+}
+
+# get any options
+while getopts ":r:b:si:h" arg; do
+    case "${arg}" in
+        r)
+          GIT_REPO=${OPTARG}
+        ;;
+        b)
+          GIT_BRANCH=${OPTARG}
+        ;;
+        s)
+          SKIP_DEPS=true
+        ;;
+        h)
+          echo "SlowMovie Install/Upgrade Script"
+          echo "Use this to install or upgrade your SlowMovie player"
+          echo "Advanced Usage:"
+          echo "[-r] specify a repo url"
+          echo "[-b] specify a repo branch"
+          echo "[-s] to skip dependency installs (update SlowMovie code only)"
+          exit 0
+        ;;
+    esac
+done
+
+# set the local directory
+LOCAL_DIR="/home/pi/$(basename $GIT_REPO)"
+
+# clear screen
+  clear;
+
+  # Set color of logo
+  # Logo generated from: http://patorjk.com/software/taag/
+  tput setaf 4
+  tput bold
+
+  cat << "EOF"
+     _____ _               __  __            _
+    / ____| |             |  \/  |          (_)
+   | (___ | | _____      _| \  / | _____   ___  ___
+    \___ \| |/ _ \ \ /\ / / |\/| |/ _ \ \ / / |/ _ \
+    ____) | | (_) \ V  V /| |  | | (_) \ V /| |  __/
+   |_____/|_|\___/ \_/\_/ |_|  |_|\___/ \_/ |_|\___|
+
+EOF
+
+# reset terminal color
+tput sgr 0
+
+echo -e "SlowMovie Repo set to $(tput setaf 3)${GIT_REPO}/${GIT_BRANCH}$(tput sgr 0)"
+echo -e "Setting up in local directory $(tput setaf 3)${LOCAL_DIR}$(tput sgr 0)"
+echo -e ""
+cd /home/pi/
+
+echo "1 - Install/Upgrade SlowMovie"
+echo "2 - Install SlowMovie Service"
+echo "3 - Uninstall SlowMovie Service"
+echo "4 - Exit"
+echo ""
+
+INSTALL_OPTION=0
+
+while [ "$INSTALL_OPTION" = "" ] || [ $INSTALL_OPTION -le 0 ] || [ $INSTALL_OPTION -gt 4 ]; do
+        read -p "Choose what you want to do? " INSTALL_OPTION
+done
```

Comment:
```suggestion
while [[ "$INSTALL_OPTION" != "" ]] && [[ $INSTALL_OPTION != [1-4] ]]; do
        read -p "Choose what you want to do? " INSTALL_OPTION
done

: ${INSTALL_OPTION:=1}
```
This will default to 1 if left blank. Using `-le` and `-gt` was also causing it to crash if anything but numbers were entered.

Reply:
Not really needed anymore with the `whiptail` menus